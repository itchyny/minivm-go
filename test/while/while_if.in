a = 10
while a >= 0
  print a
  if a > 5
    print 10
  else
    print 0
  end
  a = a - 1
end
