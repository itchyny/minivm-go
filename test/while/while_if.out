10
10
9
10
8
10
7
10
6
10
5
0
4
0
3
0
2
0
1
0
0
0
